mlflow/R/mlflow/R/tracking-runs.R

```diff
@@ -256,10 +256,14 @@ mlflow_search_runs <- function(filter = NULL,
                                run_view_type = c("ACTIVE_ONLY", "DELETED_ONLY", "ALL"), experiment_ids = NULL,
                                client = NULL) {
   experiment_ids <- resolve_experiment_id(experiment_ids)
+  # If we get back a single experiment ID, e.g. the active experiment ID, convert it to a list
+  if (is.atomic(experiment_ids)) {
```

Comment:
Technically this would allow user to pass in a single experiment id instead of a list. I think that's ok, but we should probably change the docstring above? 

Or keep the expected parameter as list and only call resolve it it has not been passed?

Reply:
Ah yeah I'll update the docstring & test to clarify that a single experiment ID is ok